app/_data/docs_nav_gateway_3.5.x.yml

```diff
@@ -228,6 +228,8 @@ items:
         url: /production/logging
       - text: Configure a gRPC service
         url: /production/configuring-a-grpc-service
+      - text: Debug Requests
```

Comment:
❗ For reviewers: I couldn't think of a great place for this to go. Does this work?

Reply:
I wonder if we should put Logging and Debug together in their own section? See how Mulesoft does it: https://docs.mulesoft.com/mule-runtime/4.5/logging-and-debugging